### ISSUE TITLE
[BUG] got this error after installing in windows, then clicking Create Project

### ISSUE THREAD

**Issue by daryledesilva** [TiagoSilvaPereira/vemto-issues#14]
```
TypeError: Cannot read property 'show' of undefined
    at app://./js/index.eb546034.js:1:24818
```

---

vemto 0.4.15
Electron 8.5.0
win32 10.0.18362
Locale: en-GB

**Comment by TiagoSilvaPereira** [comment]
Hi @daryledesilva thank you for reporting... is it always happening when you open the app?

**Comment by daryledesilva** [comment]
hey, last time I remember, it was only during post-installation then clicking Create project. after that i havent tried creating new projects again, so only saw this error once

**Comment by TiagoSilvaPereira** [comment]
Fixed on 0.5.2
